src/container.rs

```diff
@@ -386,7 +386,7 @@ impl<'docker> Container<'docker> {
             .append_pair("path", &path.to_string_lossy())
             .finish();
 
-        let mime = "application/x-tar".parse::<Mime>().unwrap();
+        let mime = "application/x-tar".parse::<Mime>()?;
```

Comment:
Can this actually ever fail? `unwrap`/`expect` seems to be ok here.

Reply:
Yes, this is right. Thanks for your advice a lot